doc/1.4/language.md

```diff
@@ -3254,6 +3259,38 @@ will give a compile error unless it appears in one of the following contexts:
   resulting object reference, the corresponding index variable of
   the object array will have an unknown value.
 
+### Method References as Function Pointers
+It is possible to retrieve a function pointer for a method by using the prefix
+operator `&` with a reference to that method. The methods this is possible with
+are subject to the same restrictions as with the [`export` object
+statement](#export-declarations): it's not possible to retrieve a function
+pointer to any inline method, shared method, method that throws, method with
+more than one return argument, or method declared inside an object array.
+
+For example, with the following method in DML:
+```
+method my_method(int x) { ... }
+```
+
+then the expression `&my_method` will be a function pointer of type:
+```
+void (*)(conf_object_t *, int);
+```
+
+The `conf_object_t *` parameter corresponds to the device instance, and is
+omitted when the referenced method is [independent](#independent-methods).
+
+Note that due to the precedence rules of `&`, if you want to immediately call a
+method reference converted to a function pointer, then you will need to wrap
+parentheses around the converted method reference. An example of where this may
+be useful is in order to call a non-independent method from within an
+independent method:
+```
+independent method my_independent_method(conf_object_t *obj, int i) {
+    (&my_method)(obj, i);
+}
+```
```

Comment:
Maybe explicitly encourage the use of this pattern for callbacks, by presenting a concrete fictional use case where `void *aux` is mapped to conf_object_t, and maybe even mention that this ensures the `state-change` notifier is invoked correctly?

Reply:
Also, please add a test case to `1.4/misc/T_notify_state.dml`

Reply:
What's the BKM for calling a C method from python? I don't want to use attributes as they would trigger state change (although if necessary I could account for that).

Reply:
Going with the notifier route.